crates/db_views_actor/src/person_view.rs

```diff
@@ -210,7 +210,7 @@ mod tests {
     let bob = Person::create(pool, &bob_form).await?;
     let bob_local_user_form = LocalUserInsertForm::builder()
       .person_id(bob.id)
-      .password_encrypted(String::new())
+      .password_encrypted(Some(String::new()))
```

Comment:
You can probably use `None` for these to make it a bit shorter (or remove the line entirely).